org.openmw.OpenMW.json

```diff
@@ -0,0 +1,272 @@
+{
+  "app-id": "org.openmw.OpenMW",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openmw-launcher",
+  "rename-appdata-file": "openmw.appdata.xml",
+  "rename-desktop-file": "openmw.desktop",
+  "finish-args": [
+    "--share=ipc", "--socket=x11", "--device=dri",
```

Comment:
I suppose that it needs full filesystem access to select Morrowind files?

Reply:
If the user is planning on installing Morrowind from original discs, then the application won't need to access anything.

For those with local installs already though, my thought was to mention the use of something like `flatpak override --filesystem=~/game/morrowind:ro org.openmw.OpenMW` in the readme - and maybe even add an entry about it on the installation wizard when built as a flatpak package.  
But yeah, maybe just giving access to the filesystem would work okay too.

Reply:
So I might be misunderstanding something, but with how the things are now, files need to be accessible somewhere at $HOME, as mount points aren't exposed unless `--filesystem=host` is passed.

Reply:
This is not a problem per se, but less technical users might be unhappy about having to fiddle with this from terminal (and there is no support for modifying permissions in GNOME Software yet). I'd pass `host` for now and if possible, add notice in the wizard you mentioned in future releases.